stb_image_write.h

```diff
@@ -1056,7 +1063,7 @@ static void stbiw__encode_png_line(unsigned char *pixels, int stride_bytes, int
    }
 
    // first loop isn't optimized since it's just one pixel    
-   for (i = 0; i < n; ++i) {
+   for (i = 0; i < n*Bpc; ++i) {
```

Comment:
I don't know much about the png format, as such I can't comment on the quality of this code (even though at first glance it looks solid). The only minor thing that I found is that the `Bpc` parameter does not follow the naming convention of other variables/parameters. See for example the function `stbi__load_main`, it also has a `bpc` parameter with a small b.

Reply:
Thanks for reviewing. The reason for the B is exactly to differentiate it from bpc. bpc means bits per component, while Bpc means bytes per components.

I didn't want bpc/8 everywhere this is used, and also didn't want bytes_per_component everywhere.
Suggestions for better naming welcome.

Reply:
Updated pull request to have comp_bytes instead of Bpc. Purpose is to follow other namings, such as stride_bytes.